vision/unit_tests/test_coordinate_lengths.py

```diff
@@ -0,0 +1,151 @@
+"""
+Testing vision.deskew.coordinate_lengths.py
+"""
+
+import unittest
+
+import numpy as np
+
+from vision.deskew.coordinate_lengths import latitude_length
+from vision.deskew.coordinate_lengths import longitude_length
+
+
+class TestCoordinates(unittest.TestCase):
+    """
+    Testing the coordinate lengths module
+    """
+
+    # Testing latitude_length
+
+    # Tests the maximum possible latitude in degrees as an input to the latitude_length function
+    def test_latitude_length_at_max_val(self) -> None:
+        """
+        Asserts that latitude_length returns expected value at maximum input (90 degrees)
+
+        Returns
+        -------
+        None
+        """
+        max_input_in_degrees = 90
+        max_input_in_radians: float = np.deg2rad(max_input_in_degrees)
+        latitude_length_at_max_latitude: float = (
+            111132.92
+            - 559.82 * np.cos(2 * max_input_in_radians)
+            + 1.175 * np.cos(4 * max_input_in_radians)
+            - 0.0023 * np.cos(6 * max_input_in_radians)
+        )
+        function_return_at_max_latitude = latitude_length(max_input_in_degrees)
+        # Tests to insure the expected output is the same as the actual output, given the same input
+        self.assertEqual(latitude_length_at_max_latitude, function_return_at_max_latitude)
+
+    # Tests the minimum possible latitude in degrees as an input to the latitude_length function
+    def test_latitude_length_min_val(self) -> None:
+        """
+        Asserts that latitude_length returns expected value at minimum input (-90 degrees)
+
+        Returns
+        -------
+        None
+        """
+        max_input_in_degrees = -90
+        min_input_in_radians: float = np.deg2rad(max_input_in_degrees)
```

Comment:
Same thing back up here. This is the min value, so `max_input_in_degrees` is misnamed